src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -318,23 +298,17 @@ const styles = StyleSheet.create({
     zIndex: 99,
     backgroundColor: COLORS.transparent
   }),
-  search: {
-    width: '100%',
-    position: 'absolute',
-    zIndex: 99,
-    backgroundColor: COLORS.transparent,
-    bottom: 0
-  },
   containerAction: {
     flexDirection: 'row',
     justifyContent: 'center',
     alignItems: 'center',
-    paddingRight: dimen.normalizeDimen(20)
+    paddingRight: dimen.normalizeDimen(20),
+    alignSelf: 'center'
   },
-  shareIconStyle: {},
-  searchContainerStyle: {
-    position: 'relative',
-    marginBottom: 0
+  searchBar: {
+    width: '100%',
+    borderBottomWidth: 1,
+    borderBottomColor: COLORS.gray210
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've made several changes to styles here. As long as these changes are intentional and don't disrupt the existing layout, they look good. However, ensure that these changes have been thoroughly tested across different screen sizes and devices.


<!-- This is an auto-generated comment by OSS CodeRabbit -->